bigquery/samples/tests/test_get_dataset.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .. import create_dataset
+from .. import get_dataset
+
+
+def test_get_dataset(capsys, client, random_dataset_id):
```

Comment:
Ditto. I recommend you use the `dataset_id` fixture instead of `random_dataset_id`, since the `dataset_id` fixture will create a dataset for you (to then get in the sample).

Reply:
This is now updated.